### PR TITLE
PA POC

### PR DESCRIPTION
**Pull request by petrjasek**
there will be changes implemented for PA POC, to be merged later on.
- [x] topics
- [x] slugline in content api
- [x] snap articles
- [ ] featured image
- [ ] new editor

### CODE REVIEW

File: server/pa/topics.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    :param filepath: path to topics file
+    """
+    topics = {}
+    if filepath is None:
+        filepath = TOPICS_FILE
+    with open(filepath, 'r') as f:
+        for topic_name in f:
+            topic_name = topic_name.strip()
+            topic_code = 'patopic:%s' % topic_name.upper()
+            if len(topic_name):
```
Comment (sivakuna-aap) [inline server/pa/topics.py]: `len(topic_name)` is redundant here. If `topic_name` is of length 0 then if evaluates it to false.

File: server/pa/topics_test.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+
+import unittest
+from .topics import get_topics
+
+
+class TopicTestCase(unittest.TestCase):
+
+    def test_get_topics(self):
+        topics = get_topics()
+        self.assertEqual(321, len(topics))
```
Comment (sivakuna-aap) [inline server/pa/topics_test.py]: We need to change the values every time we add/remove a topic. Can it be generic here?

File: server/pa/topics.txt [hunk 3]
```diff
@@ -0,0 +1,322 @@
+Accident
```
Comment (amagdas) [inline server/pa/topics.txt]: Should this actually be done using `vocabularies`?

Comment (petrjasek) [inline server/pa/topics.txt]: eventually yes. this is temporary solution

**Comment by petrjasek** [comment]
good points @sivakuna-aap will update

**Comment by ioanpocol** [comment]
PR https://github.com/superdesk/superdesk/pull/1408 has fixed the e2e tests problems.
Please rebase.

**Comment by petrjasek** [comment]
not going to merge this one..

**Pull request closed without merging**